### DISCUSSION TITLE
Which versions allow third party applications to run together with SEB?

### DISCUSSION THREAD

**Discussion by seteahconnoc** [SafeExamBrowser/seb-mac#276] [General]
Hi,
this is regarding this thread from may 2021: https://sourceforge.net/p/seb/discussion/844843/thread/96056bde7b/
I'm just wondering, if there are any news. I already read the specifications, and wanted to reassure myself. Background information: I am looking for a system, in which users with windows devices and iPads both can use the third party software "TI Nspire CAS" in the same exam environment.

---
Which versions allow third party applications to run together with SEB? safeexambrowser.org states that this feature is supported for Windows and macOS. How about iOS? Thanks in advance.

**Comment by danschlet** [comment]
Unfortunately nothing changed since then. Currently you can only run third party applications together with the Windows version of SEB. Apple though introduced an option to the macOS Assessment Mode (AAC) for allowing third party applications in macOS 10.12 Monterey, I will add that option to the SEB Mac version sometimes this year (there are some higher priority tasks first, so I can't estimate a time frame for that right now).

On iOS the AAC kiosk/single app mode doesn't offer such an option to allow third party apps. If you're using managed iPads, you could potentially activate only SEB and some specific other app. But you would need someone with iOS deployment experience to find out if this really works and to properly configure such an environment. We never tried that and don't have resources right now for evaluating such a use case.

**Comment by seteahconnoc** [comment]
Thanks for the quick and informative response.

BTW (for readers of this thread): I am aware, that the mentioned third party software "TI Nspire CAS" offers a stand-alone exam mode on it's own (both for Windows and iOS). But that's not the point that I am referring to.

The institution, I am working at, uses the concept "bring your own device (BYOD)" with Windows convertibles and iPads. I hope that I will get to know SafeExamBrowser soon. It seems to be an attractive BYOD solution. So, I'll keep SEB in mind.

**Comment by danschlet** [comment]
See https://github.com/SafeExamBrowser/seb-mac/discussions/418
